tests/integration/test_nginx_route.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# pylint: disable=redefined-outer-name,unused-argument,duplicate-code
```

Comment:
Are all these needed?

Reply:
I just refactored the code so it wouldn't be needed any more.